addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -126,17 +126,18 @@ options.registry.Rating = options.Class.extend({
      */
     _renderIcons: function () {
         const icons = {
-            'fa-star': 'fa-star-o',
-            'fa-thumbs-up': 'fa-thumbs-o-up',
-            'fa-circle': 'fa-circle-o',
-            'fa-square': 'fa-square-o',
-            'fa-heart': 'fa-heart-o'
+            'fa-star': 'fa-star',
+            'fa-thumbs-up': 'fa-thumbs-up',
+            'fa-circle': 'fa-circle',
+            'fa-square': 'fa-square',
+            'fa-heart': 'fa-heart'
         };
         const faClassActiveIcons = (this.iconType === "custom") ? this.faClassActiveCustomIcons : 'fa ' + this.iconType;
         const faClassInactiveIcons = (this.iconType === "custom") ? this.faClassInactiveCustomIcons : 'fa ' + icons[this.iconType];
         const $activeIcons = this.$target.find('.s_rating_active_icons > i');
         const $inactiveIcons = this.$target.find('.s_rating_inactive_icons > i');
         $activeIcons.removeClass().addClass(faClassActiveIcons);
         $inactiveIcons.removeClass().addClass(faClassInactiveIcons);
+        this.$target[0].setAttribute("aria-label", `${this.nbActiveIcons} out of ${this.nbTotalIcons} stars`);
```

Comment:
Nice, almost there 👍 
Unfortunately this implementation wouldn't take in account other languages, but it's easy to address ;) 
You should import the `translation` module from `core`...

```js
import { _t } from "@web/core/l10n/translation";
```

... and use it when you need to output the string

```js
_t("Hello %s", chgo_name);
```

You can see it in action in `/web/static/src/core/file_upload/file_upload_progress_record.js`, but in a lot of other modules as well ;)